safe/test/test_translations.py

```diff
@@ -136,6 +76,50 @@ def test_qgis_translations(self):
         message = 'expected %s but got %s' % (expected_message, real_message)
         self.assertEqual(expected_message, real_message, message)
 
+    @unittest.skipIf(
+        os.environ.get('ON_TRAVIS', False),
+        'Travis recognize QgsApplication as a pyqtWrapperType object.')
+    def test_qgis_app_locale(self):
+        """Test for qgis app locale."""
+
+        from safe.definitions.constants import no_field
+        self.assertEqual(no_field, u'No Field')
+
+        # run qgis on bahasa
+        _ = get_qgis_app('id', INASAFE_TEST)
+
+        # test if string from inasafe module are translated
+        from safe.definitions.constants import no_field
+        self.assertNotEqual(no_field, u'No Field')
+
+        expected_locale = 'id'
+        self.assertEqual(locale(INASAFE_TEST), expected_locale)
+
+        # check for bahasa translation
+        expected_message = (
+            'Tidak ada informasi gaya yang ditemukan pada lapisan %s')
+        real_message = QCoreApplication.translate(
```

Comment:
Why not test the `tr` function?

Reply:
no reason, I'm just applying the old unittest which is already been there before. I agree it's more usefull to test the `tr` function it self